calico/openstack/mech_calico.py

```diff
@@ -607,6 +621,11 @@ def felix_heartbeat_thread(self, hostname):
                 else:
                     LOG.error("Unexpected response from Felix on %s: %s"
                               % (hostname, rsp))
+            except Again:
+                LOG.error("No response from Felix within allowed time (%sms)" %
+                          HEARTBEAT_RESPONSE_TIMEOUT)
```

Comment:
No, because there is no caller who'd be interested in that - we're on a dedicated heartbeat thread here.